portfolio/src/test/java/DatastoreTest.java

```diff
@@ -0,0 +1,95 @@
+import static com.google.appengine.api.datastore.FetchOptions.Builder.withLimit;
+import static org.junit.Assert.assertEquals;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import org.junit.Assert;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/*TEST CLASSES AND THERE FILES HAVE TO FOLLOW ANY OF THE FOLLOWING NAMING CONVENTIONS DUE TO MAVEN
+[rest of name]Test.java -> BlahBlahBlahTest.java
+Test[rest of name].java -> TestBlahBlahBlah.java
+*/
+
+public class DatastoreTest {
+
+  // Maximum eventual consistency. [https://en.wikipedia.org/wiki/Eventual_consistency] TL;DR, some writes will commit but will always fail to apply, meaning we can't see them with global queries, strong consistency queries like ancestor queries will work
+  private final LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig()
+          .setDefaultHighRepJobPolicyUnappliedJobPercentage(100));
+
+    @Before
+    public void setUp() {
+        helper.setUp();
+    }
+
+    @After
+    public void tearDown() {
+        helper.tearDown();
+    }
+
+    //make sure datastore service starts
+    @Test
+    public void DatastoreServiceStarts(){
+        DatastoreService ds = DatastoreServiceFactory.getDatastoreService();
+        assertEquals(true, ds != null);
+    }
+
+    //Test to make sure that entities are properly posted into datastore (Checked with entities key)
+    @Test
+    public void EntitiesArePutInDatastore(){
+        DatastoreService ds = DatastoreServiceFactory.getDatastoreService();
+        Entity dummy = new Entity("Dummy");
```

Comment:
Suggestion: Define a static final variable TEST_DUMMY_NAME = "Dummy", and then reference it here.